servicestack-client/index.d.ts

```diff
@@ -0,0 +1,155 @@
+// Type definitions for servicestack-client v0.0.17
+// Project: https://servicestack.net/
+// Definitions by: Demis Bellot <https://github.com/mythz/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../isomorphic-fetch/index.d.ts" />
+
+declare namespace ServiceStack {
+export interface IReturnVoid {
+    createResponse(): any;
+}
+export interface IReturn<T> {
+    createResponse(): T;
+}
+export class ResponseStatus {
+    errorCode: string;
+    message: string;
+    stackTrace: string;
+    errors: ResponseError[];
+    meta: {
+        [index: string]: string;
+    };
+}
+export class ResponseError {
+    errorCode: string;
+    fieldName: string;
+    message: string;
+    meta: {
+        [index: string]: string;
+    };
+}
+export class ErrorResponse {
+    responseStatus: ResponseStatus;
+}
+export interface ISseCommand {
+    userId: string;
+    displayName: string;
+    channels: string;
+    profileUrl: string;
+}
+export interface ISseHeartbeat extends ISseCommand {
```

Comment:
Each of these interfaces (`ISseCommand`, `ISseHeartbeat`, `ISseJoin`, etc.) are structurally assignable to each other, just as a heads up. Not sure if it matters all that much though.